### PR TITLE
Added health data project notebook for showroom

### PR DESCRIPTION
**Pull request by olinguyen**
SQL files are added in shogun-data

### CODE REVIEW

**Comment by karlnapf** [comment]
hi @olinguyen 

I will go through this now.
First thing: could you remove all cell output from the commit? We are currently not merging notebooks with cell outputs.
However, could you paste a version *with* the outputs on github gist and paste the link (so I can see how it looks?)

File: doc/ipython-notebooks/data_project/health_data_project.ipynb [hunk 1]
```diff
@@ -0,0 +1,2248 @@
...
+    "\n",
+    "### Objective\n",
+    "\n",
+    "In the [previous notebook](https://github.com/olinguyen/gsoc2017-shogun-dataproject/blob/master/Exploring MIMIC.ipynb), we were introduced to the MIMIC database to better understand the dataset and visualize its contents. In this notebook, we'll dive into the data analysis component of this project. Using the MIMIC database which includes vital signs, medications, diagnostic code and many more variables, along with feature engineering techniques, I investigated whether I could build a robust classifier to perform the following tasks which can be used to help hospitals and care units make decisions about the management of resources and staffing:\n",
```
Comment (karlnapf) [inline doc/ipython-notebooks/data_project/health_data_project.ipynb]: We usually don't put link to external notebooks in ours. 
Maybe we can either remove this, or merge some parts of it?

File: doc/ipython-notebooks/data_project/health_data_project.ipynb [hunk 2]
```diff
@@ -0,0 +1,2248 @@
+{
+ "cells": [
```
Comment (karlnapf) [inline doc/ipython-notebooks/data_project/health_data_project.ipynb]: Notebook is long, what about a TOC?
See the GP notebooks

File: doc/ipython-notebooks/data_project/health_data_project.ipynb [hunk 3]
```diff
@@ -0,0 +1,2248 @@
...
+    "import matplotlib.pyplot as plt\n",
+    "import psycopg2\n",
+    "import getpass\n",
+    "from scipy import interp\n",
+    "from shogun import *\n",
+    "from scripts import utils\n",
+    "import os\n",
+    "import time\n",
+    "\n",
+    "SHOGUN_DATA_DIR = os.getenv('SHOGUN_DATA_DIR', '../../../data')\n",
```
Comment (karlnapf) [inline doc/ipython-notebooks/data_project/health_data_project.ipynb]: is this how we do the data path these days?

File: doc/ipython-notebooks/data_project/health_data_project.ipynb [hunk 4]
```diff
@@ -0,0 +1,2248 @@
...
...i0pm0GpTfe+89AO6//34mTZrUMh4IBPj1r3/Nm2++\nCcCCBQv48pe/3LIUW6Q7MI0ezFvPE1j7LjvG/5hAZDzTrognItHBppO1rDtey80jU8hJigh3qSIi\nIiIicoFaDcqnTp2iX79+p4VkAIfDwcKFC9mxYwcpKSkKydLthLa9h3nxaUxtFftmP0yNrQ8TpkUT\nn+ig1hvkya0l5CRFcOOw5HCXKiIiIiIin0GrQbmhoYGMjIyzHuvZsycAOTk5CsnSbZhAAPPS05g1\nS6BPf47e8AsK86MZPCKC9F5OAP5nRwn1viD/eXlvHDb93xARERER6YxaDcrGGByOsx/+aDwiQstK\npXsw9bWEnnwcDu3DmncDxZNv5tCmJnr1cdJ/iBuAnYX1rDlWy03Dk8lO1P8NEREREZHO6pLtoyzS\nVZmCk4T+8AhUV2B95dvUDZnJ7tV1JCTZGTUhCsuyaPSH+NPWYjLjXNw0XEuuRUREREQ6s3MG5T17\n9vDwww9f8HHLsvjpT3968dWJhJnZs5XQX/4bIiKwPfAo/syBbFteh9NpMeGyaOz25uXVz+8to7Qh\nwC+vyMKpLtciIiIiIp3aOYNydXU11dXVn/m4SGdljMEsfQXzxnOQlYPt6z+ExGT2ve+hqdFw2eUx\nREQ2B+LD5Y28c6iKzw1IYGiPqDBXLiIiIiIiF6vVoHzPPfe0Zx0iHYbxeTHP/h6zdT3WxBlYt30D\ny+Um/7iPwlN+Bo+IICG5+b+OP2j4w5ZiEiMc3DomNcyVi4iIiIjIpdBqUJ41a1Y7liHSMZiGotx3\nhwAAIABJREFUekK/+SmcPIL1f27FuvLzWJaFpyHEvp0eklLs9B/sbjn/9Q8qOFHt5YczexHltIex\nchERERERuVTUzEvkE8zKN+FEHrZ7f4g1enLzWMiwe0sDGBgzKQrrX9s+5dd4eXFfBdOyYpmUGRvO\nskVERERE5BL61KC8d+9eNm/eTHl5OQ6Hg+zsbObMmUNKSkp71CfSbozPi1m7FEZNbAnJAEcOe6ko\nCzJ6YiRRMc2zxiFj+OOWYiIcFneOTwtXySIiIiIi0gbOGZT//Oc/s2rVqtPGduzYwdtvv833vvc9\nRo4c2abFibQns3kt1Ndiu+K6lrGaqiC5+5pIz3SSme1qGX/3w2oOljXyjcnpJERqY
```
Comment (karlnapf) [inline doc/ipython-notebooks/data_project/health_data_project.ipynb]: pls comment here that you are using a pre-written sql statement that is long and messy :)

**Comment by lgoetz** [comment]
Hi @olinguyen, before we merge the PR, could you please address the feedback in my email (29 August) - I can see some changes, but there's still things to be done: k-fold cross validation section, figure captions and justifications for missing value replacements. Thanks!

**Comment by karlnapf** [comment]
Hi @olinguyen 
Shall we get this one in soon? Would be great to have it

**Pull request closed without merging**